core/sile.lua

```diff
@@ -245,7 +245,7 @@ end
 function SILE.call(command, options, content)
   options = options or {}
   content = content or {}
-  if SILE.traceback and not content.lno then
+  if SILE.traceback and not (type(content) == "table" and content.lno) then
```

Comment:
Hey @Darkyenus this is a case where I thought your code was overly complex and I turned out to be wrong and had to backtrack. I'm sorry I had to work through all this  and get to where you got the hard way --- but I hope I did improve things along the way too.